win/pkgs/ort.ps1

```diff
@@ -40,6 +40,11 @@ $update_gtest = $true
 $update_onnx = $false
 $update_protobuf = $false
 $use_bat = $false
+$run_tests = $true
+if ($ENV:ROASTER_ORT_SKIP_TESTS)
```

Comment:
1. ${Env:ROASTER_ORT_SKIP_TESTS}
2. There's no point to use both env var and var.
3. Please use auto detection instead. i.e. with nvidia-smi and nvidia-smi -L

Reply:
Fixed in latest update.

The option is detected using:
`$((Get-Command -Name "nvidia-smi" -ErrorAction SilentlyContinue) -and (nvidia-smi -L))`